test/templates/alert/index.njk

```diff
@@ -71,6 +71,71 @@
 
 <br><br>
 
+<h1>alert --no-icon</h1>
+
+<!-- output the component example and the code snippet -->
+{% set example %}
+{{  alert(
+        properties={
+            text: '<p class="fd-alert__text">Integer turpis augue, tincidunt dignissim mauris id, rhoncus dapibus purus. Maecenas et enim odio. Nullam massa metus, varius quis vehicula sed, pharetra mollis erat. Sed bibendum, sapien at posuere interdum, libero est sollicitudin magna, ac gravida tellus purus eu ipsum. Proin ut quam arcu. <a href="#" class="fd-link">Learn More</a></p>',
```

Comment:
The `<a href="#" class="fd-link">Learn More</a>` links should be removed from here (multiple places on this page) as the playground is serving as a style reference specific to alerts and the self-contained styles playground will not handle these as those styles come packaged with the link component.  The main docs can still use links as those are more use case-based examples.

Reply:
done